swarm/src/protocols_handler/mod.rs

```diff
@@ -171,9 +170,8 @@ pub trait ProtocolsHandler {
     /// Should behave like `Stream::poll()`.
     ///
     /// Returning an error will close the connection to the remote.
```

Comment:
```suggestion
```

`ProtocolsHandlerEvent::Close` got introduced, thus this comment is obsolete.

https://github.com/libp2p/rust-libp2p/pull/1328/files?file-filters%5B%5D=.rs&hide-deleted-files=true#diff-7329c74601d5a2cab9c980cf8fea8031R321